libs/form-builder/src/lib/types/field.ts

```diff
@@ -1,6 +1,14 @@
+import { FieldTranslationId } from '@postybirb/types';
 import { PrimitiveRecord } from './primitive-record';
 
-export type FieldType<T extends PrimitiveRecord, V, F extends string> = {
+// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries, @typescript-eslint/no-unused-vars, import/order
+import { type fieldLabelTranslations } from 'apps/postybirb-ui/src/components/form/fields/field-translations';
```

Comment:
This is why enfore-boundaries warns in all other places. I just wanted to add @link to the comment so users who want to expand label can just jump into the fieldLabelTranslations and modify them

Reply:
This will need to be documented when the guide is written for onboarding other devs.

Reply:
I will add exception to the rule globally i guess and add comment regarding this in the eslintrc.js

Reply:
done